spec/models.spec/users.spec.js

```diff
@@ -37,10 +37,42 @@ describe('UserSchema', function() {
         expect(info.name).toBe('Kingsley');
         expect(info.emailAddress).toBe('chitech247@gmial.com');
         });
-    });*/
+    });
      it('should throw an error when data is not sufficient, name or emailAddress', function() {
         User.create({name: 'Kingsley'}, function(err, info){
         expect(err).toBeDefined();
         });
     });
+   it('should find users', function() {
+        User.find(function(err, info){
+            if(err){
+                expect(err).toBeDefined();
+            }
+        var type = typeof info;
+        expect(type).toEqual('object');
+        expect(info.length).toBeGreaterThan(99);
```

Comment:
this is not testing anything, can we assure that info.length will ever be less than 99?

you can rewrite it and test in another way